### PR TITLE
Jc progress bar

### PR DESCRIPTION
**Pull request by jcode89**
Adds the tqdm module to `requirements.txt` and adds the functionality to display a progress bar on the command line to both rethink and sqlite.

### CODE REVIEW

**Comment by jcode89** [comment]
Addresses issue #23

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+coverage==4.0.3
+-e git+https://github.com/jcode89/DataDotPy.git@664703f1c7c7d00e65e9214aa868e9dc6c32d2bb#egg=data_py-jc-progress-bar
```
Comment (jamalhansen) [inline requirements.txt]: We should probably remove this line.

**Comment by jamalhansen** [comment]
Overall, looks good.  How does the status bar look?

File: requirements.txt (hunk 1 above)
Comment (jcode89) [inline requirements.txt]: Okay removed

**Comment by jcode89** [comment]
close #23

**Pull request merged**